<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDataset</class>
 <widget class="QDialog" name="SelectDataset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelAllDataset">
           <property name="text">
            <string>&amp;Select  dataset:</string>
           </property>
           <property name="buddy">
            <cstring>listWidgetAllDataset</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidgetAllDataset"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelDatasetToMerge">
           <property name="text">
            <string>&amp;Dataset to merge:</string>
           </property>
           <property name="buddy">
            <cstring>listWidgetSelected</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidgetSelected"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButtonUp">
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonNext">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRemove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDown">
           <property name="text">
            <string>Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelMergedName">
         <property name="text">
          <string>&amp;Dataset Names (separated by ;):</string>
         </property>
         <property name="buddy">
          <cstring>lineEditMergedName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditMergedName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOk">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidgetAllDataset</tabstop>
  <tabstop>listWidgetSelected</tabstop>
  <tabstop>lineEditMergedName</tabstop>
  <tabstop>pushButtonUp</tabstop>
  <tabstop>pushButtonRemove</tabstop>
  <tabstop>pushButtonDown</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>SelectDataset</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
